integration/relayer/cmd_relayer_test.go

```diff
@@ -290,93 +231,6 @@ func TestBlogIBC(t *testing.T) {
 		)),
 	))
 
-	env.Must(env.Exec("create a post type list in an IBC module",
```

Comment:
This was only scaffolded here and is not been tested. We already have a complete test in  the apps repository

Reply:
I think we should keep it. It the scaffolding part doesn't work it is important to know. Testing it here may make sense too as the relayer app won't test the latest ignite app